native/src/modules/common/renderHtml.js

```diff
@@ -147,6 +147,10 @@ const renderHtml = (html: string, files: PageResourceCacheStateType, theme: Them
         list-style-position: inside;
         \`} */
     }
+    
+    body * {
+        font-size: ${theme.fonts.contentFontSize};
```

Comment:
Doesn't this set the font-size of every header ( "h1, h2" etc), element?

Reply:
Fixed